src/middlewared/middlewared/plugins/apps/logs.py

```diff
@@ -38,8 +39,8 @@ def __init__(self, *args, **kwargs):
 
     def validate_log_args(self, app_name, container_id):
         app = self.middleware.call_sync('app.get_instance', app_name)
-        if app['state'] not in ('RUNNING', 'DEPLOYING'):
-            raise CallError(f'App "{app_name}" is not running')
+        if AppState(app['state']) not in (AppState.CRASHED, AppState.RUNNING, AppState.DEPLOYING):
```

Comment:
Same here, 3 separate instances of the AppState class

Reply:
Fixed it ✅ (also it would just be 1 instance of the class as the other are declared values)